internal/cli/root.go

```diff
@@ -67,7 +67,7 @@ func Execute() {
 		cli.renderer.Errorf(err.Error())
 
 		instrumentation.ReportException(err)
-		os.Exit(1)
+		os.Exit(1) // nolint:gocritic
```

Comment:
Upon calling `os.Exit()` we immediately exit and all deferred funcs will not run. However we do need to use a defer over here to be able to recover from panics, so we're just gonna skip linting this part.